writing/inclusive-language.md

```diff
@@ -28,7 +28,7 @@ Stereotyping means presuming a range of things about people based on their perso
 
 Language matters. It remains a common practice in IT to use terminology like slave/master or blacklist/whitelist that reference slavery or segregation.
 
-There's always a better way to label the relationship between two entities. In the case of databases, choosing more descriptive terms like replica/primary, secondary/primary, follower/leader or standby/active instead of slave/master not only removes the reference to slavery, but also [makes it easier to understand](https://twitter.com/geeksam/status/1038081079641395201) for people not familiar with the terminology.
+There's always a better way to label the relationship between two entities. In the case of databases, choosing more descriptive terms like replica/primary, secondary/primary, follower/leader or standby/active instead of slave/master not only removes the reference to slavery, but also makes it easier to understand for people not familiar with the terminology.
```

Comment:
Do you think it's worth linking back in some way to [our writing house style](https://github.com/springernature/frontend-playbook/blob/main/writing/house-style.md#plain-english), which explicitly says we should "Avoid metaphor, simile, and slang"? 

It is a metaphor, after all.

Reply:
Done, thank you!